scripts/import_video_from_yaml.py

```
@@ -0,0 +1,263 @@
+django.setup()
+# --- End Django Setup ---
+
+from django.db import transaction
+from endoreg_db.models import (
+    Video,
+    Patient,
+    Center,
+    Label,
+    LabelVideoSegment,
+    Gender,
+    SensitiveMeta,
```

Ruff (0.8.2)

`endoreg_db.models.SensitiveMeta` imported but unused

Remove unused import: `endoreg_db.models.SensitiveMeta`

(F401)